client/src/components/Production/ProductionModalBike.jsx

```diff
@@ -0,0 +1,169 @@
+        </FormControl>
+      </div>
+      <div style={formStyle}>
+        <FormControl style={elementStyle}>
+          <FormLabel>Chain</FormLabel>
+          <Select ref={chainRef}>
+            {isSuccessPart &&
+              dataPart.data
+                .filter((part) => part.type === 'chain')
+                .map((part) => (
+                  <Fragment key={part._id}>
+                    <option value={part._id}>{part.name}</option>
+                  </Fragment>
+                ))}
+          </Select>
+        </FormControl>
+        <FormControl style={elementStyle}>
+          <FormLabel>Frame</FormLabel>
+          <Select ref={frameRef}>
+            {isSuccessPart &&
+              dataPart.data
+                .filter((part) => part.type === 'frame')
+                .map((part) => (
+                  <Fragment key={part._id}>
+                    <option value={part._id}>{part.name}</option>
+                  </Fragment>
+                ))}
+          </Select>
+        </FormControl>
+        <FormControl style={elementStyle}>
+          <FormLabel>Pedal</FormLabel>
+          <Select ref={pedalRef}>
+            {isSuccessPart &&
+              dataPart.data
+                .filter((part) => part.type === 'pedal')
+                .map((part) => (
+                  <Fragment key={part._id}>
+                    <option value={part._id}>{part.name}</option>
+                  </Fragment>
+                ))}
+          </Select>
+        </FormControl>
+      </div>
+      <div style={formStyle}>
+        <FormControl style={elementStyle}>
+          <FormLabel>Brakes</FormLabel>
+          <Select ref={brakesRef}>
+            {isSuccessPart &&
+              dataPart.data
+                .filter((part) => part.type === 'brakes')
+                .map((part) => (
+                  <Fragment key={part._id}>
+                    <option value={part._id}>{part.name}</option>
+                  </Fragment>
+                ))}
+          </Select>
+        </FormControl>
+        <FormControl style={elementStyle}>
+          <FormLabel>Seat</FormLabel>
+          <Select ref={seatRef}>
+            {isSuccessPart &&
+              dataPart.data
+                .filter((part) => part.type === 'seat')
+                .map((part) => (
+                  <Fragment key={part._id}>
+                    <option value={part._id}>{part.name}</option>
+                  </Fragment>
+                ))}
+          </Select>
+        </FormControl>
+        <FormControl style={elementStyle}>
+          <FormLabel>Fork</FormLabel>
+          <Select ref={forkRef}>
+            {isSuccessPart &&
+              dataPart.data
+                .filter((part) => part.type === 'fork')
+                .map((part) => (
+                  <Fragment key={part._id}>
+                    <option value={part._id}>{part.name}</option>
+                  </Fragment>
+                ))}
+          </Select>
+        </FormControl>
```

Comment:
should all of these fields be required to make a bike?

Reply:
You don’t need pedals for your bike? @DavidLiang01

Reply:
Yes, all these are required to make a bike. 👍

Reply:
Both in real life and how the backend was defined!